src/components/molecules/Visualizer/Widget/Storytelling/index.tsx

```diff
@@ -174,22 +194,23 @@ const Title = styled(Text)`
   }
 `;
 
-const StyledIcon = styled(Icon)`
-  color: ${props => props.theme.main.text};
+const StyledIcon = styled(Icon)<{ publishedTheme: PublishTheme }>`
```

Comment:
Sorry, one more thing. In components like this StyledIcon where all you are using is mainIcon from publishedTheme, can you pass JUST what you need, rather than the whole publishedTheme object.
In this case you could pass a `color=publishedTheme.mainIcon` prop instead

Reply:
This wasn't addressed. Please let me know if you don't catch what I mean and I can explain more!